connect/file/src/test/java/org/apache/kafka/connect/file/FileStreamSourceTaskTest.java

```diff
@@ -204,8 +210,6 @@ private void writeTimesAndFlush(OutputStream os, int times, byte[] line) throws
 
     @Test
     public void testMissingFile() throws InterruptedException {
```

Comment:
Can we use this opportunity to improve the name of this test? `testMissingFile` sounds like we're testing an error case while this is testing using stdin instead of file.

Reply:
Yeah, done!